cmd/libs/go2idl/parser/parse_test.go

```diff
@@ -119,8 +127,12 @@ type FooObject interface {
 	CommonObject
 }
 
+
+func proto.AFunc( proto.Object proto.Object ) proto.Frobber {}
```

Comment:
The space before the ")" comes from the separator between parameters, i.e.,
```
{{range $s.Parameters}}{{Raw .}} {{end}}  //note the space before {{end}}
```
I can't find an elegant way to remove the space after the last parameter. 

For symmetry, I added the space after "(".